server/models/historical_meteorological.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Float, Integer, String
+
+from server.db.base_class import Base
+
+
+class Historical_Meteorological(Base):
+    __tablename__ = "Historical Meteorological Data"
```

Comment:
Out of curiosity, do you mind explaining the name choice? Are you anticipating something like "Forecast Meteorological Data" in the future?

If it's all the same to you, I'd like to set the `__tablename__` to "meteorological_data" since historical is implied in my mind. When making these changes, please update all other model and schemas classes too.

As always, I'm open to discuss! Let me know what you think.

---

### SQL Naming Conventions

Unfortunately, I can't find a clear answer on what should be done. However, what I do know is that consistency is more important above all else. So whatever we do, it should be mirrored.

Therefore, it's always good practice to see how our database currently looks (e.g., patterns in table naming, etc).

You'll notice our table names are all written in lowercase with spaces represented by a single underscore and pluralized (e.g., `buoys`).

It's also worth noting that we must make sure any deviation from the aforementioned pattern is compliant with `sqlalchemy` (our ORM).